src/pages/report/MyCardPage.tsx

```
@@ -1,5 +1,61 @@
+import KPISearchBar from "../../components/report/KPISearchBar";
+import ReportNavbar from "../../components/report/ReportNavbar";
+
 const MyCardPage = () => {
-  return <div>MyCardPage</div>;
+  const [cardCount] = useState(10);
+
+  const loaderRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    const observer = new IntersectionObserver(() => {
+      threshold : 1
```

ESLint

[error] 'threshold:' is defined but never used.

(no-unused-labels)